app/src/main/java/com/pavesid/androidacademy/services/MoviesWorkerRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.pavesid.androidacademy.services
+
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequestBuilder
+import com.pavesid.androidacademy.BuildConfig
+import java.util.concurrent.TimeUnit
+
+object MoviesWorkerRepository {
```

Comment:
it seems like it is a factory for `request`. Or you can just put it into hilt. But it does not look like Repository